MicrosoftFluentUI.podspec

```diff
@@ -298,6 +298,14 @@ fi', :execution_position => :before_compile }
     tableview_ios.source_files = ["ios/FluentUI/Table View/**/*.{swift,h}"]
   end
 
+  s.subspec 'TextField_ios' do |textfield_ios|
+    textfield_ios.platform = :ios
+    textfield_ios.dependency 'MicrosoftFluentUI/Button_ios'
+    textfield_ios.dependency 'MicrosoftFluentUI/Label_ios'
+    textfield_ios.dependency 'MicrosoftFluentUI/Separator_ios'
```

Comment:
if we have a clear icon, shouldn't we have xcfilelist to make sure subspec still works?

Reply:
I might have put the asset in the wrong place, but `pod lib lint` didn't have any issues. The image is in the FluentUI_ios asset catalog like the other images.